source/assets/js/helpers/mediaqueries.js

```diff
@@ -52,10 +57,66 @@
 		}
 	});
 
+	function getBreakpointValue(breakpoint) {
+		var value = 0;
+
+		if (typeof breakpoints[breakpoint] !== 'undefined') {
+			value = parseInt(breakpoints[breakpoint], 10);
+		} else {
+			throw 'Breakpoint not found: "' + breakpoint + '"';
+		}
+		return value;
+	}
+
+	function mediaQuery(options) {
```

Comment:
I'd prefer to call this function `query` since it's not a media query in the common sense.